### ISSUE TITLE
Spring Boot 3.x support

### ISSUE THREAD

**Issue by Mantas330BJ** [Azure/azure-servicebus-jms#35]
Are there any plans to support Spring Boot 3.x?
Particularly, migrating from javax.jms to jakarta.jms.

**Comment by EldertGrootenboer** [comment]
Thank you for suggesting this feature. We have opened an investigation task for this in our backlog, and will update this issue when we have more information. To help us get this prioritized, it would be helpful to see others vote and support this feature, as well as explain their scenarios.

**Comment by ksalazar-91** [comment]
Yes we will be supporting Jakarta https://github.com/Azure/azure-servicebus-jms/pull/42 . Pending release
